src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const fee = 40;
```

Comment:
You should change this variable name to something more understandable

Reply:
fixed